### DISCUSSION TITLE
希望增加微信朋友圈分享生成图文链接

### DISCUSSION THREAD

**Discussion by lyqscmy** [geeknote-net/feedback#14] [Ideas]
在微信浏览器中打开geeknote的文章后，点分享到朋友圈是以文字分享的URL。

希望能生成图文链接。

不太懂前端，似乎是要加入weixin的SDK，提供摘要和图片？

**Comment by chloerei** [comment]
做了一些研究，似乎是需要 SDK，而且需要注册开发者账号和服务端签名。这侵入性太强了，其他社交平台都是设置好 meta tag 就行。

需要 SDK 的方式暂时不想支持，如果有更好的方法会考虑实现。

提示：从系统浏览器分享到微信是会显示摘要的。
